Example/Example/Purchase/PurchaseLogicController.swift

```diff
@@ -118,11 +123,97 @@ final class PurchaseLogicController {
     }
   }
 
+  func payWithCashApp() {
+    if cashRequest != nil {
+      paykit?.authorizeCustomerRequest(cashRequest!)
+    }
+  }
+
+  private var cashButton: CashAppPayButton?
+
+  private var cashRequest: CustomerRequest?
+
+  static var cashData: CashAppSigningData?
+
+  private lazy var paykit: PayKit? = {
+    guard let clientId = Afterpay.cashAppClientId else {
+      assertionFailure("Couldn't get cash app client id")
+      return nil
+    }
+
+    let paykitSdkEnv = Afterpay.environment == .production ? PayKit.Endpoint.production : PayKit.Endpoint.sandbox
+    let sdk = PayKit(clientID: clientId, endpoint: paykitSdkEnv)
+    sdk.addObserver(self)
+
+    return sdk
+  }()
+
+  func enableCashButton() {
+    cashButton?.alpha = 1
+    cashButton?.isEnabled = true
+  }
+
+  func disableCashButton() {
+    cashButton?.isEnabled = false
+    cashButton?.alpha = 0.3
```

Comment:
We really should provide this in the SDK, I will prompt design on this one but this is fine for now

Reply:
I have made this change and pushed out a release. Please update to 0.2.0 and you can remove the alpha!

Reply:
Updated in [ce37b12](https://github.com/afterpay/sdk-ios/pull/243/commits/ce37b12ea106378d96564f214a542848e55aee58)